TestResultSummaryService/parsers/TestBenchmarkParser.js

```diff
@@ -41,7 +33,8 @@ class TestBenchmarkParser extends Parser {
             for(let result of results) {
                 if (this.canParse(result.testName)) {
                     let {benchmarkName, benchmarkVariant} = this.getBenchmarkInfo(result.testName);
-                    const {testData} = Utils.parseOutput(benchmarkName, benchmarkVariant, result.testOutput);
+                    const benchmarkParserKey = Utils.getBenchmarkParserKey(result.testName);
```

Comment:
There is no need to call `canParse()` in line 34. It means calling getBenchmarkParserKey() twice. We can just use line 36.

```
const benchmarkParserKey = Utils.getBenchmarkParserKey(result.testName);
if (benchmarkParserKey) {
   let {benchmarkName, benchmarkVariant} = this.getBenchmarkInfo(result.testName);
   ...
}
```